comfy/utils.py

```diff
@@ -407,3 +407,19 @@ def update_absolute(self, value, total=None, preview=None):
 
     def update(self, value):
         self.update_absolute(self.current + value)
+
+def launch_custom_web_browser(args, address, port):
+    import webbrowser
+    custom_browser = webbrowser.GenericBrowser(args.custom_browser_path)
+    
+    #If the user has supplied custom browser args
+    if args.custom_browser_args_csv:
+
+        #args.browser_args_csv is a csv, split it into a list, prepend "--" to each arg and append to custom_browser.args
```

Comment:
Is there a reason to do this whole thing, vs just `--custom-browser-args "--incognito --etc --etc"`?

Reply:
In my experience the parser couldn't handle it, but I'm not a python developer